flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/source/row/converter/JDBCRowConverter.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.api.java.io.jdbc.source.row.converter;
+
+import org.apache.flink.types.Row;
+
+import java.io.Serializable;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+/**
+ * Convert row from JDBC result set to a Flink row.
+ */
+public interface JDBCRowConverter extends Serializable {
+
+	/**
+	 * Set {@link Row} with data retrieved from {@link ResultSet}.
+	 *
+	 * @param resultSet ResultSet from JDBC
+	 * @param reuse The row to set
+	 */
+	Row setRow(ResultSet resultSet, Row reuse) throws SQLException;
```

Comment:
What about to call the method `convert`? It's not very explict that it happens some conversion in `setRow`.